doc/ref_model/chapters/chapter04.rst

```diff
@@ -269,62 +269,62 @@ or events should be collected and monitored by monitoring tools.
 |         | Infrastructure      |           |                                                                          |
 |         | Capability          |           |                                                                          |
 +=========+=====================+===========+==========================================================================+
-|i.pm.001 | Host CPU usage      |nanoseconds| Per Compute node. It maps to ETSI GS NFV-TST 008 V3.5.1 [5] clause 6,    |
+|i.pm.001 | Host CPU usage      |nanoseconds| Per Compute node. It maps to ETSI GS NFV-TST 008 V3.5.1 :cite:p:`etsigsnfvtst008` clause 6,    |
```

Comment:
I think this breaks the grid tables. You need to fit the text to the grid. 
This comment is valid for all similar cases.

Reply:
I double checked and yes, this breaks the tables and RM does not compile. 
As you might also recognized the automatic builds are not working anymore, so I would like to ask the sub-project leads to ensure that the proposed changes are not breaking the build before they get merged. Thanks.